javascript/src/test/pages/SearhHistory/AllUser.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { act, render } from '@testing-library/react';
+import AllUser from 'main/pages/SearchHistory/AllUser';
+
+describe('All Users tests', () => {
+    act(()=>{
+        test('renders without errors', () => {
+            render(<AllUser />);
+        });
+    });   
```

Comment:
I don't think the test should be wrapped in an act block. I don't believe jest will run this test if you wrap it like this.